src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -16,6 +16,9 @@
 // ***************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
 //  SVE Intrinsics
 
+// Sve
+HARDWARE_INTRINSIC(Sve,           LoadVector,                                                       -1,      2,      true,  {INS_sve_ld1b,       INS_sve_ld1b,       INS_sve_ld1h,       INS_sve_ld1h,       INS_sve_ld1w,       INS_sve_ld1w,       INS_sve_ld1d,       INS_sve_ld1d,       INS_sve_ld1w,       INS_sve_ld1d},    HW_Category_MemoryLoad,            HW_Flag_Scalable|HW_Flag_MaskedOperation)
+HARDWARE_INTRINSIC(Sve,           TrueMask,                                                         -1,      1,      false, {INS_invalid,        INS_sve_ptrue,      INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_EnumPattern,           HW_Flag_Scalable|HW_Flag_HasImmediateOperand|HW_Flag_ReturnsPerElementMask)
```

Comment:
This should have `NoCodeGen` flag as well

Reply:
On the other hand, why not setting `INS_sve_ptrue` here?

Reply:
Only `byte` type is valid for `TrueMask`, and it is there in the table.

(however, I'm expecting `TrueMask` to expand to more types in the API review)

Reply:
ah, i missed that.